testarch/devices/clutff-unidir-s4/CMakeLists.txt

```diff
@@ -1,8 +1,17 @@
 add_file_target(FILE arch.xml SCANNER_TYPE xml)
 
+get_target_property_required(PYTHON3 env PYTHON3)
+get_target_property(PYTHON3_TARGET env PYTHON3_TARGET)
+
+set(UPDATE_TILES "${symbiflow-arch-defs_SOURCE_DIR}/utils/update_arch_tiles.py")
+set(UPDATE_TILES_CMD "${PYTHON3} ${UPDATE_TILES} --in_xml /dev/stdin --out_xml /dev/stdout")
+set(UPDATE_TILES_DEPS "")
```

Comment:
At a minimum you still want to depend on the script itself

Reply:
Done